### ISSUE TITLE
Suspending Julia via Ctrl-Z requires two presses

### ISSUE THREAD

**Issue by fingolfin** [JuliaLang/julia#49224]
The following happens for me with Julia 1.6.7, 1.8.5 and 1.9.0rc1 on both macOS 12.6.4 and a Ubuntu 20.04.

On Unix like systems (including macOS and Linux) one can usually suspend programs with a REPL using Ctrl-Z and then later resume them (e.g. by entering `fg` in bash).

This kinda works for Julia, but one has to press ctrl-Z twice: after the first press, Julia is in a weird limbo state: it echos whatever I type, even overdrawing its own REPL prompt, but is otherwise unresponsive. Ctrl-C is also ignored. Pressing ctrl-Z a second time then fully suspends it and I am back in my shell.

Also, if I press Ctrl-Z Ctrl-C Ctrl-Z and then resume it (via `fg`), it crashes.

**Comment by mbaz** [comment]
I can't reproduce this on Linux; ctrl-z has always worked as expected.

**Comment by thofma** [comment]
Can reproduce it with the same versions, both on Linux and macOS.

**Comment by giordano** [comment]
@fingolfin are you using juliaup?  I can reproduce if using the juliaup launcher, not if using the real julia executable directly.

**Comment by fingolfin** [comment]
@giordano indeed, I do! that makes sense, I guess. so perhaps this should be transferred to the juliaup issue tracker?
